include/exadg/functions_and_boundary_conditions/evaluate_functions.h

```diff
@@ -189,18 +189,18 @@ struct FunctionEvaluator<1, dim, Number>
 
   static inline DEAL_II_ALWAYS_INLINE //
     dealii::Tensor<1, dim, dealii::VectorizedArray<Number>>
-    value(FunctionCached<1, dim> const & function,
-          unsigned int const             face,
-          unsigned int const             q,
-          unsigned int const             quad_index)
+    value(ContainerInterfaceData<1, dim, double> const & function,
+          unsigned int const                             face,
+          unsigned int const                             q,
+          unsigned int const                             quad_index)
   {
     dealii::Tensor<1, dim, dealii::VectorizedArray<Number>> value;
 
     std::array<dealii::Tensor<1, dim, Number>, dealii::VectorizedArray<Number>::size()>
       tensor_array;
 
     for(unsigned int v = 0; v < dealii::VectorizedArray<Number>::size(); ++v)
-      tensor_array[v] = function.tensor_value(face, q, v, quad_index);
+      tensor_array[v] = function.get_data(face, q, v, quad_index);
```

Comment:
Here, the order of arguments has to be changed when changing the function name. The same for the two other calls below.